runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -219,25 +226,47 @@ public void onLookupMenuClicked(PlayerMenuOptionClicked event)
 	{
 		if (event.getMenuOption().equals(LOOKUP))
 		{
-			executor.execute(() ->
+			lookupPlayer(Text.removeTags(event.getMenuTarget()));
+		}
+	}
+
+	private void lookupPlayer(String playerName)
```

Comment:
Try not to mix subscribers and defined functions. This should be defined below the subscriber methods.